script/check-quality-gate.sh

```diff
@@ -43,21 +43,24 @@ until [[ ${status} != "PENDING" && ${status} != "IN_PROGRESS" ]]; do
     task="$(curl --location --location-trusted --max-redirs 10 --silent --fail --show-error --user "${SONAR_TOKEN}": "${ceTaskUrl}")"
     status="$(jq -r '.task.status' <<< "$task")"
 done
+printf '\n'
 
 analysisId="$(jq -r '.task.analysisId' <<< "${task}")"
 qualityGateUrl="${serverUrl}/api/qualitygates/project_status?analysisId=${analysisId}"
 qualityGateStatus="$(curl --location --location-trusted --max-redirs 10 --silent --fail --show-error --user "${SONAR_TOKEN}": "${qualityGateUrl}" | jq -r '.projectStatus.status')"
 
-printf '\n'
+dashboardUrl="$(sed -n 's/dashboardUrl=\(.*\)/\1/p' "${metadataFile}")"
+analysisResultMsg="Detailed information can be found at: ${dashboardUrl}\n"
+
 if [[ ${qualityGateStatus} == "OK" ]]; then
    set_output "quality-gate-status" "PASSED"
    success "Quality Gate has PASSED."
 elif [[ ${qualityGateStatus} == "WARN" ]]; then
    set_output "quality-gate-status" "WARN"
-   warn "Warnings on Quality Gate."
+   warn "Warnings on Quality Gate.${reset}\n\n${analysisResultMsg}"
```

Comment:
What's the point of using `${reset}` ?

Reply:
This is to reset text formatting in the UI

Without the `${reset}`:
```diff
! ✖ Warnings on Quality Gate.

! Detailed  information can be found at: http://localhost:9000/dashboard?id=project&branch=master
---
- ✖ Quality Gate has FAILED.

- Detailed  information can be found at: http://localhost:9000/dashboard?id=project&branch=master 
```

With the `${reset}`:
```diff
! ✖ Warnings on Quality Gate.

Detailed  information can be found at: http://localhost:9000/dashboard?id=project&branch=master
---
- ✖ Quality Gate has FAILED.

Detailed  information can be found at: http://localhost:9000/dashboard?id=project&branch=master 
```

I can remove it if you want the whole output to be yellow/red

Reply:
Thanks for clarifying. Make sense to clear colors.